homeassistant/components/google_drive/const.py

```diff
@@ -2,4 +2,9 @@
 
 from __future__ import annotations
 
+from datetime import timedelta
+
 DOMAIN = "google_drive"
+
+SCAN_INTERVAL = timedelta(minutes=5)
```

Comment:
This seems too frequent. The sensors are only expected to be updated after a backup (typically daily or weekly) or after a manual delete (uncommon). How about going with every few hours? Whoever wants more frequent can call the homeassistant.update_entity action.